dashboard/src2/components/NavigationItems.vue

```diff
@@ -118,6 +124,7 @@ export default {
 		}
 	},
 	mounted() {
+		this.$session.roles.fetch();
```

Comment:
fetching roles in NavigationItems component doesn't feel right, if you have fetch it once then do it in main.js